CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * Add neighboring dropdown menu to Flaw Description for its review workflow (`OSIDB-2623`)
 * Add CVE Require Description for AdvancedSearch (`OSIDB-2624`)
 * Support for references and acknowledgements on flaw creation (`OSIDB-2319`)
+* Sort Advanced Search Options (`OSIDB-2805`)
```

Comment:
Minor suggestion to inform about what sorting has been implemented
```suggestion
* Sort Advanced Search Options alphabetically (`OSIDB-2805`)
```

Reply:
Thanks, I will update